CMakeLists.txt

```diff
@@ -33,8 +33,7 @@ if(ENABLE_FASTCGI)
   list(APPEND THIRD_PARTY_MODULES thrift)
 endif()
 
-# Add bundled libafdt if the system one will not be used
-if(NOT LIBAFDT_LIBRARY)
+if (NOT MSVC)
```

Comment:
This looks incorrect, both for whitespace, and if there is a shared libafdt available on a non-windows platform